fakewalletreact/android/build.gradle

```diff
@@ -20,4 +20,8 @@ buildscript {
     }
 }
 
+plugins {
```

Comment:
switch to project version catalog so this is always in sync. this fixes CI errors (plugin already defined on classpath with different version) when the versions are out of sync